q.js

```diff
@@ -91,6 +91,7 @@ var nextTick =(function () {
     var flushing = false;
     var requestTick = void 0;
     var isNodeJS = false;
+    var laterQueue = []; // queue for late tasks
```

Comment:
Yeah, this should definitely look differently in v2 :)

I'll try to amend this PR Friday to fire the browser hooks - let me know if
there is anything else you'd like me to change. If you feel like
implementing this yourself differently then you're of course also welcome.

On Wed, Feb 11, 2015 at 7:20 AM, Kris Kowal <notifications@github.com>
wrote:

> In q.js <https://github.com/kriskowal/q/pull/643#discussion_r24477631>:
>
> > @@ -91,6 +91,7 @@ var nextTick =(function () {
> >      var flushing = false;
> >      var requestTick = void 0;
> >      var isNodeJS = false;
> > +    var laterQueue = []; // queue for late tasks
>
> You are right that minimum viable is appropriate for this. My intuition is
> that layering queues would compose better and be more maintainable in the
> long run. I am content to make v2 the long run branch though, if there’s a
> pragmatic solution here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kriskowal/q/pull/643/files#r24477631>.
>